modern/package.json

```diff
@@ -50,10 +51,17 @@
     ]
   },
   "devDependencies": {
+    "@parcel/packager-raw-url": "^2.6.0",
+    "@parcel/transformer-webmanifest": "^2.6.0",
     "eslint": "^8.16.0",
     "eslint-config-airbnb": "^19.0.4",
     "eslint-plugin-import": "^2.26.0",
     "eslint-plugin-react": "^7.30.0",
-    "eslint-plugin-react-hooks": "^4.5.0"
+    "eslint-plugin-react-hooks": "^4.5.0",
+    "os-browserify": "^0.3.0",
+    "parcel": "^2.6.0",
+    "path-browserify": "^1.0.1",
+    "process": "^0.11.10",
+    "rimraf": "^3.0.2",
```

Comment:
I have a clarification here about `browserify`, `process` and `rimraf`. It's something to do with node, but can you please explain why we need it in our app. We don't use any node features in the app itself, as far as I know.

Reply:
It seems these polyfills are needed for some packages of Parcel. It's only for development.

Reply:
If they are needed for Parcel why it doesn't depend on them? A bit weird.

Reply:
https://parceljs.org/features/node-emulation/#polyfilling-%26-excluding-builtin-node-modules

Reply:
If I correctly understand the link you provided above, we don't need them. What happens without them? What is not working? I don't really understand adding dependencies without fully understanding why you need them. If you think they are really needed, please provide proper explanation.

Reply:
In the development and migration, I had occurred to some errors that required these polyfills. Not in the main project, but the packages and Parcel. I will test them on my Linux and Windows machine and notify you with the results.

Reply:
OK, I see it wants to install `process`. What about other?

Reply:
> > rimraf, I using this package before build to clean the last build for production
> 
> What's the problem without it? Why can't you build directly?

On my Windows pc, I occurred to some cache problems, and I decided to use Rimraf. We can remove this package and use the built-in support for this feature in the parcel if you like.

Reply:
> Why can't you build directly?

I think other polyfills are installed for Dotenv because it uses os and path modules in Nodejs. We can remove them and install them in the future if needed.

Reply:
Sounds good. Let's remove anything that's not needed immediately and add as we go.